cmd/nibirud/cmd/airdrop.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/genutil"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
-	ibctransfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	ibctransfertypes "github.com/cosmos/ibc-go/v2/modules/apps/transfer/types"
```

Comment:
just downgrading to v2, code dose not affected by this.
Because the v2 module name is same as v3
V2: https://github.com/cosmos/ibc-go/blob/v2.2.0/modules/apps/transfer/types/keys.go#L12
V3: https://github.com/cosmos/ibc-go/blob/v3.0.0/modules/apps/transfer/types/keys.go#L12